chap-intro.tex

```diff
@@ -533,44 +504,15 @@ \subsection{An Architecture-Specific Mapping into 64-bit MIPS}
   capability stuff)}, and RISC: CHERI's features make possible
 the implementation of a wide variety of OS, compiler, and application-originated policies on a common protection substrate that optimizes fast paths through hardware support.
 
-%
-% Comment out this text in order to de-emphasize a coprocessor interpretation
-% of CHERI.  However, we might want to lift some ideas from this paragraph and
-% weave them back into the overall text here at some point.
-%
-%The capability coprocessor is a coprocessor in two senses.
-%First, the capability coprocessor occupies a portion of the existing ISA
-%encoding dedicated to extensions (typically referred to as coprocessor
-%instructions).
-%Second, the capability coprocessor supplements the general-purpose integer register
-%file with its own ISA-managed registers, as well as performing (and
-%transforming) memory access, and delivering exceptions to the main pipeline,
-%requiring hardware resources that interact with the primary processor
-%pipeline.  This behavior is comparable in many ways to system,
-%floating-point, vector, or cryptographic coprocessors that will similarly
-%supplement the base ISA and processor features.
-
-
-%\psnote{rephrasing, as we've already introduced the fact that we're
-%prototyping on MIPS}
-%In order to prototype this approach, we have instantiated our ideas about CHERI
-%capability access to a specific instruction set: the 64-bit MIPS ISA.
-%This
-Our prototype of this approach,  instantiating our ideas about CHERI
-capability access to a specific instruction set (the 64-bit MIPS ISA)
-has necessarily led to a set of congruent implementation decisions about
-register-file size, selection of specific instructions, exception handling,
-memory alignment requirements, and so on, that reflect that starting-point
-ISA.
-These decisions might be made differently with another starting-point ISA as
-they are simply surface features of the underlying approach; we anticipate that
-adaptations to ISAs such as ARM, RISC-V, and x86-64 would adopt
-instruction-encoding conventions, and so on, more in keeping with their
-specific flavor and design (see Chapters~\ref{chap:cheri-riscv}
-and~\ref{chap:cheri-x86-64}).
-
-Other design decisions reflect the goal of creating a platform for prototyping
-and exploring the design space itself.
+CHERI-RISC-V applies CHERI protections in different ways compared to
+our initial CHERI-MIPS architecture.  Some of these differences arise
+from differences in the base ISA; we anticipate that
+adaptations of CHERI to ISAs will adopt instruction-encoding
```

Comment:
```suggestion
adaptations of CHERI to ISAs will adopt local instruction-encoding
```
It to me like there's a missing qualifier to "instruction-encoding conventions"

Reply:
The qualifier is later in the sentence but it is perhaps a bit awkard to read.  Namely, the "in keeping with their specific flavor and design." at the end.  Maybe I should reword it as:

```
we anticipate that adaptations of CHERI to ISAs will adopt conventions such as instruction-encoding in keeping with their specific flavor and design.
```